### DISCUSSION TITLE
Is Explorer Patcher Win versions

### DISCUSSION THREAD

**Discussion by WonderWoman1977** [valinet/ExplorerPatcher#3263] [General]
Hey,

I am on W11 Pro on 2 of my computers and several Hyper-v VMs (not activated).  Will EP work for W11 Pro or is it only for W10 versions?

Thanx.......

**Comment by pyrates999** [comment]
It works in windows 11 pro as far as I know.  Just use releases of windows 11 and not developer versions.

Also, in the settings app, untick "Get the latest updates as soon as available" so that preview updates are not automatically installed.

All preview updates are then shown as download and install and are never automatically installed if you have the above unticked.

Preview updates are also not fully tested by Microsoft and thus could have bugs.

You should only be installing preview updates if you are a developer or are OK with running pre-release versions of EP.

**Comment by christopbader** [comment]
I have 5 machines that all run Win-11 Pro with EP Version 65.3,  there are no issues with EP and all seem to run well, They are stable releases of Win-11 none are developer versions
![Screenshot (65)](https://github.com/valinet/ExplorerPatcher/assets/109197478/f7b7a6d2-70f3-40a2-aac0-de8ddf75be9c)
![Screenshot (66)](https://github.com/valinet/ExplorerPatcher/assets/109197478/97035e32-9ecb-474e-86f6-325c4ffd787e)

**Comment by WonderWoman1977** [comment]
Once it is installed, how can I see what version I am running?  Is there a separate EP that I should install for Win11 Pro?

**Reply by christopbader** [reply]
What ever version you have should work on Win-11 Pro just make sure you have the latest version, the version number is on the 'About' section of EP
![Screenshot (67)](https://github.com/valinet/ExplorerPatcher/assets/109197478/b3e3e6d7-7bbd-49c7-a267-dee4cabe6c41)
